vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/overview/components/ProjectCollection.java

```diff
@@ -0,0 +1,191 @@
+    projectSearchField.setValueChangeMode(ValueChangeMode.LAZY);
+    projectSearchField.addValueChangeListener(event -> {
+      projectPreviewFilter = event.getValue().trim();
+      projectPreviewGridLazyDataView.refreshAll();
+    });
+  }
+
+  private void configureProjectCreationButton() {
+    createProjectButton.addClickListener(listener -> {
+      fireClickEvent();
+    });
+  }
+
+  private void layoutSearchField() {
+    projectSearchField.setPlaceholder("Search");
+    projectSearchField.setClearButtonVisible(true);
+    projectSearchField.setSuffixComponent(VaadinIcon.SEARCH.create());
+    projectSearchField.addClassNames("search-field");
+  }
+
+  private void layoutProjectCreationButton() {
+    createProjectButton.addClassName("add-project-button");
+  }
+
+  private void layoutGrid() {
+    projectGrid.addColumn(new ComponentRenderer<>(
+        item -> new Anchor(String.format(Projects.PROJECT_INFO, item.projectId().value()),
+            item.projectCode()))).setHeader("Code").setWidth("7em").setFlexGrow(0);
+
+    projectGrid.addColumn(new ComponentRenderer<>(
+        item -> new Anchor(String.format(Projects.PROJECT_INFO, item.projectId().value()),
+            item.projectTitle()))).setHeader("Title").setKey("projectTitle");
+
+    projectGrid.addColumn(new LocalDateTimeRenderer<>(
+        projectPreview -> asClientLocalDateTime(projectPreview.lastModified()),
+        "yyyy-MM-dd HH:mm:ss")).setKey("lastModified").setHeader("Last Modified");
+
+    projectGrid.setMultiSort(true);
+  }
+
+  private void layoutTitleSection() {
+    titleSection.addClassName("title");
+    titleSection.setText(title);
+  }
+
+  private void layoutControlSection() {
+    controlSection.addClassName("controls");
+  }
+
+  private void layoutGridSection() {
+    gridSection.addClassName("projects-grid");
+  }
+
+  private void fireClickEvent() {
+    var clickedEvent = new ProjectCreationClickedEvent(this, true);
+    projectCreationClickedListeners.forEach(listener -> listener.onComponentEvent(clickedEvent));
+  }
+
+  private LocalDateTime asClientLocalDateTime(Instant instant) {
+    ZonedDateTime zonedDateTime = instant.atZone(this.clientZoneId);
+    return zonedDateTime.toLocalDateTime();
+  }
+
+  /**
+   * Create a new instance of a project collection.
+   *
+   * @param title                     the title to be displayed on top of the component
+   * @param clientZoneId              the client's zone id
+   * @param projectInformationService the project information service to enable lazy loading of
+   *                                  projects.
+   * @return
+   * @since 1.0.0
+   */
+  public static ProjectCollection create(String title, ZoneId clientZoneId,
+      ProjectInformationService projectInformationService) {
+    Objects.requireNonNull(projectInformationService);
+    Objects.requireNonNull(clientZoneId);
+    return new ProjectCollection(title, clientZoneId, projectInformationService);
+  }
```

Comment:
Since this component is specific in its usage we don't expect to change the title, therefore you can imo just hardcode it in this collection